src/pages/appflow/destinations/manual.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Manual Deploy'
+previousText: 'Command-line Deploy'
+previousUrl: '/docs/appflow/destinations/cli'
+nextText: 'Package'
+nextUrl: '/docs/appflow/package/intro'
+---
+
+To deploy an existing binary a build that has the correct security profile selected for the app store destination must be built.
+
+- From the `Deployment` page `New Deployment` should be selected.
+
+![/docs/assets/img/appflow/appflow-deploy-existing.png](/docs/assets/img/appflow/appflow-deploy-existing.png)
+
+- Select the build from the `1. Select build` page
+
+- Choose the destination for the build from the `2. Configure Deployment` page
+
+- The result will be shown in the `Deployment` page
```

Comment:
I think this needs an extra step with the list of deployments and explanation that you can click on the row to see the logs